extensions/elytron-security-jdbc/runtime/src/main/java/io/quarkus/elytron/security/jdbc/AttributeMappingConfig.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.elytron.security.jdbc;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
```

Comment:
It could be useful to expand a bit about what these properties mean here and remove a link to a WildFly class